communitiesuk/funding-service-design-application-store
# FS-3431 refactored to use sqs funcs from fsd-utils

https://dluhcdigital.atlassian.net/browse/FS-3431

### Description
Refactored to use SQS funtions from fsd-utils


 Linked PR's 
https://github.com/communitiesuk/funding-service-design-assessment-store/pull/313
https://github.com/communitiesuk/funding-service-design-docker-runner/pull/52

### How to test
- Checkout this branch and the[ assessment-store](https://github.com/communitiesuk/funding-service-design-assessment-store/pull/313) & [docker-runner](https://github.com/communitiesuk/funding-service-design-docker-runner/pull/52)
- Rebuild the containers with `--no-cache`
- Submit an application (from CYP/NSTF or COF R3w2) and observe the application automatically imported to the assessment
